pandas-input.md

```diff
@@ -259,7 +261,7 @@ One of the great things about Pandas is that we can do things with every observa
 ```python
 print(type(df['x1']))
 ```
-we find that it's a `pandas.core.series.Series` -- **series** for short.  A series is simply a dataframe with a single variable, which simplifies the bookkeeping a bit.  
+we find that it's a `pandas.core.series.Series` -- **series** for short.  A series is essentially a dataframe with a single variable or column, which simplifies the bookkeeping a bit.  
```

Comment:
Changed wording -- it was a bit misleading to say that a Series **is** a DataFrame with one column. It sounds better to me to say they are similar